images/libvirtd/Dockerfile

```diff
@@ -13,6 +13,15 @@ RUN yum install -y \
 COPY augconf /augconf
 RUN augtool -f /augconf
 
+# Faking qemu-kube
+# qemu-kube originally requires docker and sudo - also
+# access to docker service (which is running on the host)
+# modifying this image would be possible, but it's not
+# essential for the demo.
+# It should be obsoleted anyway once the virt-handler
+# populates the cgroup.
```

Comment:
Well we need to find a solution for the namespaces, until then we will probably have the script.